src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -8517,6 +8521,7 @@ namespace AirflowNetworkBalanceManager {
             Tamb = Zone(i).OutDryBulbTemp;
             CpAir = PsyCpAirFnWTdb(ZoneAirHumRatAvg(i), MAT(i));
             AirDensity = PsyRhoAirFnPbTdbW(OutBaroPress, MAT(i), ZoneAirHumRatAvg(i));
+            // TODO: InfilHeatLoss is watt?
```

Comment:
ToDo?

Reply:
@mjwitte @lgu1234 
Based on the implementation it seems the unit of InfilHeatGain should be [J] instead of [W] (as multiplied by ReportingConstant? But the comments in the header indicates it should be watt values. And I wanted to report exfiltration in watts as well.
```
    struct AirflowNetworkReportVars
    {
        // Members
...
        Real64 InfilHeatGain;      // Heat Gain {W} due to infiltration
        Real64 InfilHeatLoss;      // Heat Loss {W} due to infiltration
...
        Real64 ExfilSensiLoss;     // Sensible heat Loss rate {W} due to exfiltration
        Real64 ExfilLatentLoss;    // Latent heat Loss rate {W} due to exfiltration
```

Reply:
@XuanLuoLBL I agree these look incorrect. Searching further, it appears that these are never used. See https://github.com/NREL/EnergyPlus/issues/7162#issuecomment-461261073